lib/streamlit/elements/markdown.py

```diff
@@ -209,6 +211,27 @@ def latex(self, body: Union[SupportsStr, "sympy.Expr"]) -> "DeltaGenerator":
         latex_proto.element_type = MarkdownProto.Type.LATEX
         return self.dg._enqueue("markdown", latex_proto)
 
+    @gather_metrics("divider")
+    def divider(self) -> "DeltaGenerator":
+        """Display horizontal ruler.
+
+        Note that you can achieve the same effect with st.write("---") or
+        even just "---" in your script (via magic).
+
+        (No parameters)
```

Comment:
- The official Markdown syntax [spec](https://daringfireball.net/projects/markdown/syntax#hr) from [RFC 7763](https://www.rfc-editor.org/rfc/rfc7763.html#section-6.1.4) calls it a horizontal rule. We should change ruler to rule.
- The first line is missing an `a` before horizontal
- To add a notes section, use the `.. note::` admonition
- If there are no params, jump directly to the example after the note. No need for `(No parameters)`

```suggestion
        """Display a horizontal rule.

        .. note::
            You can achieve the same effect with st.write("---") or
            even just "---" in your script (via magic).
```

Reply:
After committing my suggestion:
![image](https://user-images.githubusercontent.com/20672874/222379063-82733293-9904-411f-b669-ee57d97ba860.png)

The current version renders like:
![image](https://user-images.githubusercontent.com/20672874/222379121-5fc8f36f-64f1-4edc-9b32-ed44bae13db4.png)

Reply:
Thanks. Accepted.